.env

```diff
@@ -20,7 +20,7 @@ REACT_APP_TYPE_AND_SWAP_URL=https://type-swap.kyberswap.com/api
 REACT_APP_TRANSAK_URL=https://global.transak.com
 REACT_APP_TRANSAK_API_KEY=48949c0b-2d20-4e3a-a311-51ca91ae8c0d
 
-REACT_APP_KS_SETTING_API=https://ks-setting.stg.kyberengineering.io/api
+REACT_APP_KS_SETTING_API=https://ks-setting.dev.kyberengineering.io/api
```

Comment:
is this intention

Reply:
update to the correct one. to consistent with others api